client/src/components/Verejne/Map/ClusterMarker/index.js

```diff
@@ -38,8 +38,9 @@ const ClusterMarker = ({
     className = cluster.numPoints === 1 ? 'simple-marker' : 'cluster-marker'
     children = cluster.numPoints !== 1 && <span className="marker__text">{cluster.numPoints}</span>
   } else {
-    //TODO: fix classnames after we api provides enough information
+    //TODO: fix classnames after the api provides enough information
     className = cluster.numPoints === 1 ? 'company-marker' : 'cluster-marker'
+    className += cluster.points[0].tradewithgovernment ? ' government' : ''
```

Comment:
There is a nice utility for this, https://github.com/JedWatson/classnames. Check it, maybe you'll like it :slightly_smiling_face:

Reply:
Let me know if you feel like using it, or just ping me to merge.

Reply:
looks great, and feels much better to use :)
pushing changes now